src/components/Dashboard/Health/HealthItem.js

```diff
@@ -3,31 +3,33 @@ import PropTypes from 'prop-types';
 import { Icon } from 'patternfly-react';
 import { get } from 'lodash';
 
+import { HEALTHY, DEGRADED, ERROR, UNKNOWN } from './strings';
+
 const HealthStatus = {
   0: {
-    message: 'OCS is Healthy',
+    message: HEALTHY,
```

Comment:
is the key always a number 0-3. Can you use an array instead?

Reply:
> nevermind

may not be needed once @rawagner suggestion is implemented

> HealthItem should receive status (OK/WARNING/ERROR) and message so it can be reused for all dashboards